### PR TITLE
Integrate classification loss modules

### PR DESCRIPTION
**Pull request by illian01**
## Description

Please include a summary in English, of the changes in this pull request. If it closes an issue, please mention it here.

Closes: #225 

You should link at least one existing issue for PR. Before your create a PR, please check to see if there is an issue for this change.  
PRs from forked repository not accepted.

## Change(s)

- `nn.CrossEntropy` works well when `pred` and `target` have same shapes.

```python
pred.shape, target.shape
>> (torch.Size([32, 3]), torch.Size([32, 3]))
nn.CrossEntropyLoss()(pred, target)
>> tensor(1.1373, device='cuda:0', grad_fn=<DivBackward1>)
torch.sum(-target * F.log_softmax(pred, dim=-1), dim=-1).mean()
>> tensor(1.1373, device='cuda:0', grad_fn=<MeanBackward0>)
```

- It works well when `label_smoothing` is applied as `0.1`.

```python
pred.shape, target.shape
>> (torch.Size([32, 3]), torch.Size([32, 3]))
nn.CrossEntropyLoss(label_smoothing=0.1)(pred, target)
>> tensor(1.1349, device='cuda:0', grad_fn=<DivBackward1>)
torch.sum(-(((1 - 0.1) * target) + (0.1 / 3)) * F.log_softmax(pred, dim=-1), dim=-1).mean()
>> tensor(1.1349, device='cuda:0', grad_fn=<MeanBackward0>)
```

- Thus, we don't need extra variant of cross-entropy loss module for classification task.
- Removed `LabelSmoothingCrossEntropy` and `SoftTargetCrossEntropy`, leave only `CrossEntropyLoss`
- Tested all classification task models and segmentation task models.

## Changelog

If you PR to `dev` branch, please add a brief summary of the change to the **Upcoming Release** section of the [`CHANGELOG.md`](https://github.com/Nota-NetsPresso/netspresso-trainer/blob/master/CHANGELOG.md) file and include a link to the PR (formatted in markdown) and a link to your github profile.

For example,

```
- Added a new feature by `@myusername` in [PR 2023](https://github.com/Nota-NetsPresso/netspresso-trainer/pull/2023)
```

## Code Formatting

If you PR to either `master` or `dev` branch, you should follow the code linting process. Please check your code with `lint_check.sh` in `./scripts` directory.
For more information, please read the contribution guide in `CONTRIBUTING.md`.

### CODE REVIEW

**Review by deepkyu — APPROVED** [review]
LGTM
Thanks for checking also with segmentation models
